webui/react/src/contexts/Store.tsx

```diff
@@ -157,6 +158,23 @@ const initInfo: DeterminedInfo = {
   version: process.env.VERSION || '',
 };
 
+export const OSSUserRole: V1Role = {
+  name: 'OSS User',
+  permissions: [
+    {
+      id: -1,
+      isGlobal: true,
+      name: 'oss_user',
```

Comment:
I wonder if we still expect a name attribute on Permissions, now that the id is changing from a number to the V1PermissionType enum / string. If everything works we can put it in a future pr/ticket but just noting this.